src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line.

Reply:
4th line is not empty